src-docs/src/views/color_picker/color_stops.js

```diff
@@ -87,6 +113,41 @@ export default () => {
           stopType="fixed"
         />
       </EuiFormRow>
+
+      <EuiFormRow label="Stepped color segments">
+        <EuiFlexGroup alignItems="center" gutterSize="xs">
+          <EuiFlexItem>
+            <EuiColorStops
+              label="Stepped color segments"
+              onChange={setSteppedColorStops}
+              colorStops={steppedColorStops}
+              stepNumber={value}
+              min={0}
+              max={100}
+              stopType="stepped"
+            />
+          </EuiFlexItem>
+
+          <EuiFlexItem grow={false}>
+            <EuiPopover
+              panelStyle={{ minWidth: 380 }}
+              button={button}
+              isOpen={isPopoverOpen}
+              closePopover={closePopover}>
+              <EuiFormRow label="Number of steps" display="columnCompressed">
+                <EuiRange
+                  value={value}
+                  onChange={(e) => setValue(e.target.value)}
```

Comment:
```suggestion
                  onChange={(e) => setValue(parseInt(e.target.value))}
```

Prevents a propType warning